css-three-column-layout/styles.css

```diff
@@ -0,0 +1,80 @@
+html {
+  font-family: Lato, sans-serif;
+  color: white;
+}
+
+section {
+  padding: 35px;
+  justify-content: center;
+  align-items: center;
+}
+
+h2 {
+  font-size: 25px;
+  font-weight: 500;
+  letter-spacing: 0.5px;
+}
+
+h3 {
+  font-weight: 500;
+}
+
+p {
+  font-size: 12px;
+  color: #cfd7dd;
+  line-height: 18px;
+}
+
+.bottom-margin-sm {
+  margin-bottom: 12px;
+}
+
+.header {
+  padding: 14px;
+}
+
+.container {
+  width: 800px;
+  margin: auto;
+  display: flex;
```

Comment:
With layout classes, `display: flex` is typically reserved for the `row` class. Try removing this and see if your layout still works as intended 🤓